sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -106,7 +106,8 @@ def plot_precision_recall_curve(estimator, X, y,
     Parameters
     ----------
     estimator : estimator instance
-        Trained classifier.
+        Fitted classifier, or a fitted pipeline in which the last estimator
```

Comment:
```suggestion
        Fitted classifier, or a fitted :class:`~sklearn.pipeline.Pipeline` in which the last estimator
```

Reply:
Great idea, thanks! I incorporated that into my latest commit.